src/lib/EventListener/ContentProxyCreateDraftListener.php

```diff
@@ -77,6 +77,8 @@ public function create(ContentProxyCreateEvent $event): void
             []
         );
 
+        $options->set('content_draft', $contentDraft);
+
         if ($options->get('isOnTheFly', false)) {
```

Comment:
I see here naming inconsistency. You provided an option with underscores, yet the existing one is CamelCase. Which one is correct?